src/HotChocolate/Spatial/src/Types/Exceptions/GeoJsonSerializationException.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace HotChocolate.Types
+{
+    public class GeoJsonSerializationException : Exception
```

Comment:
This should be based on the GraphQLException.

Reply:
Resolved in #1870